### ISSUE TITLE
Edge staircase clip is too tall

### ISSUE THREAD

**Issue by Paril** [jdolan/quetoo-data#58]
http://i.imgur.com/aoD8cO6.jpg

This second-to-bottom stair has a huge extended clip area for some reason, making it hard to step up onto it from the side. Pushing forward from screenshot does not put me up the stair as it should.

**Comment by jdolan** [comment]
<img width="1680" alt="screen shot 2016-11-12 at 8 59 18 am" src="https://cloud.githubusercontent.com/assets/643118/20238329/5764b8ee-a8b6-11e6-9dd3-a0e437193dbe.png">

There is a weird clip there. My guess is, this enables a trick jump to the crates.

**Comment by Panjoo** [comment]
Yeah, that was to enable a well-known jump up the crates from the left side, instead of using the first little crate most q2 players start their doublejump off that first step. I will take it out I guess since too much has changed in terms of movement. 
In my opinion most of the hardest jumps in the original q2 version are much too easy in quetoo, while other easy jumps are now a lot trickier to do. For instance small (32units) crate jumps are harder, while normal strafejumps will take you _a lot_ farther. Which isn't bad but feels a bit less hardcore to an inveterate q2-er like me, almost noobish:P.

[noobjumpz.zip](https://github.com/jdolan/quetoo-data/files/587422/noobjumpz.zip)
(quetoo 0.1.0 Nov 7)

BTW -offtopic- I really miss a huh-huh jump sound, which made it feel like your quakeguy was breaking a sweat to jump. It also was an important sound cue as to your location.

(PS. Although rarely, sometimes after a demo has ended quetoo just closes to desktop by itself, the bastard.)

**Comment by Paril** [comment]
I think the new trickjump code makes that extra clip redundant. I like that the skill ceiling is lower for jumping - it seems less "elitist" now. I can chalk kills down to my firing and not trickjumping now 🔫 

I thought I fixed that demo crash :( can you post a demo that causes it?

**Comment by Panjoo** [comment]
I just did:)

> I like that the skill ceiling is lower for jumping - it seems less "elitist" now. I can chalk kills down to my firing and not trickjumping now

This is both good and bad I think. Good because like you said it lowers the skill ceiling, but bad because experienced jumpers will do the otherwise hard jumps with their eyes closed and will have even more advantage than you might think. I'm okay with it as long as the original q2 jumps stay intact, albeit easier.

**Comment by Paril** [comment]
Oh I didn't even check what the zip was, my mistake

**Comment by jdolan** [comment]
I wouldn't be opposed to a subtle voiced jump noise, but I do find Q2's jump sounds kind of annoying after about 60 seconds, hehe. I haven't checked in with @anthonywebb1 for a month or two. Let's see if he writes back ;)

**Comment by jar3817** [comment]
The male jump sound has always annoyed me, that's why I always pick female
player. It's funny when my 3yr old son starts going "huh huh huh" when I'm
watching or playing...

**Comment by anthonywebb1** [comment]
@jdolan Sorry about the silence, been a bit busy on another game. Yes I can look into this. Same sort of cloth and equipment jumping sound, but with a less aggressive/quieter HUH sound?

I'll make 5 and you can always leave a couple out if you don't want the HUH sound for every single jump.

**Comment by jdolan** [comment]
Yes, that would be perfect! I think it'd be best if the voice was sort of generic -- male, but not explicitly "human." We do have another player model, Gork, who is a green troll-demon-ork thing. If you listen to the player's pain sounds (players/common/pain*), I think what I'm saying will make sense ;)

Got a link to the other game?

**Comment by Panjoo** [comment]
@Paril, 
One or two times the game closed itself after the demo ended, but it also happened when I pressed a bind to start recording. What I did was "bind s stop" and "bind d record demoname", so that could easily stop and start recording. Seemed very random but it sometimes happens that once I press the record bind to re-record the same demo it just exits the game.
Also, it sometimes happens that I have to pre-load the map before the demo even wants to load.
(now recorded with Quetoo ver. Nov 12 mingw32)
[noobjumps.zip](https://github.com/jdolan/quetoo-data/files/587764/noobjumps.zip)
